azure-pipelines.yml

```diff
@@ -423,20 +423,25 @@ jobs:
       set -ex
       trap 'get_logs' EXIT
       get_logs() {
-        SUB_PATH="hw/top_earlgrey"
-        mkdir -p "$OBJ_DIR/$SUB_PATH" "$BIN_DIR/$SUB_PATH"
-        cp -rLvt "$OBJ_DIR/$SUB_PATH/" \
+        mkdir -p $OBJ_DIR/hw/top_earlgrey/chip_earlgrey_cw310/
+        mkdir -p $BIN_DIR/hw/top_earlgrey/chip_earlgrey_cw310/
+        manifest_dir=$($REPO_TOP/bazelisk.sh cquery --output=starlark \
+          --starlark:file=ci/scripts/get-bitstream-fragment-dir.bzl \
+          //hw/bitstream/vivado:fpga_cw310_manifest)
+        echo "$($REPO_TOP/bazelisk.sh cquery --output=starlark \
+          --starlark:file=ci/scripts/get-bitstream-fragment-files-relative.bzl \
+          //hw/bitstream/vivado:fpga_cw310_manifest)" > files.txt
+        rsync --archive --copy-unsafe-links --verbose --files-from=files.txt \
+          ${manifest_dir}/ $BIN_DIR/hw/top_earlgrey/chip_earlgrey_cw310
+        rm files.txt
+        cp -rLvt $OBJ_DIR/hw/top_earlgrey/chip_earlgrey_cw310/ \
```

Comment:
I haven't looked at it deeply, but it seems like it ought to be possible. We've kept up the `ninja` directory structure for CI artifacts for a long time, but it doesn't necessarily have to persist forever. ;)

I considered starting that now, but ultimately, I didn't want to change too many things for this one PR, since it already got to hundreds of lines, haha.

Reply:
Done (pending CI verification)